Client/client.go

```diff
@@ -0,0 +1,62 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"github.com/dosarudaniel/CS438_Project/logger"
+	clientService "github.com/dosarudaniel/CS438_Project/services/client_service"
+	"google.golang.org/grpc"
+	"time"
+)
+
+// printFeature gets the feature for the given point.
+func requestFile(client clientService.ClientServiceClient, fileMetadata *clientService.FileMetadata, log *logger.Logger) error {
+	log.Info(fmt.Sprintf("Request file %v from %v", fileMetadata.FilenameAtOwner, fileMetadata.OwnersID))
+	ctx, cancel := context.WithTimeout(context.Background(), 10 * time.Second)
+	defer cancel()
+
+	response, err := client.RequestFile(ctx, fileMetadata)
+	if err != nil {
+		log.Fatal(fmt.Sprintf("%v.RequestFile(_) = _, %v: ", client, err))
+		return err
+	}
+
+	log.Info(response.Text)
+	return nil
+}
+
+func main() {
+	peersterAddress := flag.String("PeersterAddress", "127.0.0.1:5000", "Peerster address to connect to")
+	file := flag.String("file", "file1.txt", "file name at owner")
+	ownersID := flag.String("ownersID", "f98eeff24e2fced1a1336182a3e8775326262914cc4087066d9346431795ccdb", "File owner's ID")
+	nameToStore := flag.String("nameToStore", *file, "Name used to store the download file")
```

Comment:
I think we need to have empty strings as defaults for easier checking if they were provided to the client program. If the string is empty, the flag wasn't provided, so that should result in a failure. However, I have a strong wish to make our client to work as a read-evaluate-print-loop (REPL):

```
$ ./Client
Hello! You are running Peerster v2! Instructions:
to download a file: download <file_id> from <node_id>
to find successor of a node: find successor of <node_id>
> find successor of aa
successor of aa: cd
> download hh from kk
downloading...
downloaded (or failed to download due to timeout, for example)
```

Reply:
Yes, agree. 
Also, the GUI client should on our priority list.